File: solutions/beecrowd/1044/1044.py
import sys


def main():
    for line in sys.stdin:
        a, b = map(int, line.split())

        if a % b == 0 or b % a == 0:
            print('Sao Multiplos')
        else:
            print('Nao sao Multiplos')


main()

File: solutions/beecrowd/1044/1044.clj
(ns main
  (:require [clojure.string :as str]))

(defn multiple? [a b]
  (if (or (zero? (mod a b)) (zero? (mod b a)))
    (println "Sao Multiplos")
    (println "Nao sao Multiplos")))

(defn main []
  (loop [line (read-line)]
    (when line
      (let [[a b] (map #(Integer/parseInt %) (str/split line #" "))]
        (multiple? a b))
      (recur (read-line)))))

(main)
